.travis.yml

```diff
@@ -1,20 +1,21 @@
 language: c
-sudo: required
-install:
-  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
-  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docgen.sh
-script:
-  - bash -ex .travis-opam.sh
-after_success:
-  - test "$GEN_DOC" = 1 && test $TRAVIS_PULL_REQUEST = "false" && test $TRAVIS_BRANCH = "master" && bash -ex .travis-docgen.sh
+sudo: false
+services:
+  - docker
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+script: bash -ex ./.travis-docker.sh
```

Comment:
TODO: re-add doc upload once https://github.com/xapi-project/nbd/pull/71 is finalized